LLama/Exceptions/RuntimeError.cs

```diff
@@ -58,6 +58,25 @@ public LLamaDecodeError(DecodeResult returnCode)
     }
 }
 
+/// <summary>
+/// `llama_decode` return a non-zero status code
+/// </summary>
+public class MissingTemplateError
```

Comment:
```suggestion
public class MissingTemplateException
```

Exception classes should end with `Exception` ([docs](https://learn.microsoft.com/en-us/dotnet/standard/exceptions/best-practices-for-exceptions#end-exception-class-names-with-exception)).

`RuntimeError` and `LLamaDecodeError` break this rule and should be fixed sometime!